citations-api/api/src/java/org/sakaiproject/citation/api/CitationService.java

```diff
@@ -229,15 +229,15 @@
 	 */
 	public CitationCollectionOrder getNestedCollection(String citationCollectionId);
 	/**
-	 ** Removes the section of a citation collection and its children
+	 ** Removes the section of a citation collection and its children or removes a citation
 	 * @param collectionId
 	 * @param locationId
 	 */
-	public void removeSection(String collectionId, int locationId);
+	public void removeSectionOrCitation(String collectionId, int locationId);
```

Comment:
No just nested citations.  Both a nested citation and an unnested citation have a 'remove' link which goes to CitationHelperAction.doRemoveSelectedCitations(RunData) where the locationId is checked for; if it has a locationId (i.e., it's nested), it deletes the citation from the nested collection ; if it's null or 0, it deletes it from the usual unnested collection using the usual flow.